Jenkinsfile

```diff
@@ -3603,6 +3603,7 @@ assert_frame_equal(training_curve, gt_curve, rtol=1e-3, atol=1e-3)"'''
         trainer.precision=16 \
         trainer.gradient_clip_val=1.0 \
         exp_manager.exp_dir=examples/nlp/language_modeling/gpt_pretrain_results \
+        exp_manager.resume_if_exits=False \
```

Comment:
resume_if_exists*

Reply:
Thanks @titu1994, corrected the typo. Also the reason behind setting this to False is that this test aka `Megatron GPT Pretraining and Resume Training PP=2` is picking up the last checkpoint from the previous test aka `Megatron GPT with KERPLE Pretraining and Resume Training TP=2` and it errors out as it tries to find that checkpoint in the current test's `exp_dir`, but can't find it.

Reply:
This was happening bcoz I had commented these lines out by mistake in the lightning 2.0 upgrade PR: `https://github.com/NVIDIA/NeMo/blob/main/Jenkinsfile#L3583-L3584`. Fixing it ASAP in a separate PR.